src/aleph/vm/conf.py

```diff
@@ -387,6 +387,7 @@ def check(self):
             assert (
                 check_system_module("kvm_amd/parameters/sev_es") == "Y"
             ), "SEV-ES feature isn't enabled, enable it in BIOS"
+            assert is_command_available("sevctl"), "Command `sevctl` not found, run `cargo install sevctl`"
```

Comment:
Should this command be installed globally on the system or is it a standalone binary whose path should be in the settings ?

Reply:
It is supposed to be already on the system, as we commented on the sync meeting, we will create a binary to be downloaded and installed directly on the system, and also on the client side.

Reply:
In this case, we should add a setting `SEVCTL_PATH` similar to `FIRECRACKER_PATH` for binaries that are provided by our package.

Reply:
I opened a Pull Request to build and bundle the `sevctl` program as part of the Debian packages.
See https://github.com/aleph-im/aleph-vm/pull/629

This is not blocking and can be marked as resolved.